insolar/record/record.go

```diff
@@ -14,7 +14,6 @@
 // limitations under the License.
 //
 
-// /go:generate protoc -I./vendor -I./ --gogoslick_out=./ insolar/record/record.proto
```

Comment:
Why abandon go generate?

Reply:
It doesn't work, and not everyone have installed `protoc`, so preferable way is to delete this